homeassistant/components/xiaomi_miio/number.py

```diff
@@ -98,9 +109,30 @@ class XiaomiMiioNumberDescription(NumberEntityDescription):
         step=1,
         method="async_set_volume",
     ),
+    FEATURE_SET_OSCILLATION_ANGLE: XiaomiMiioNumberDescription(
+        key=ATTR_OSCILLATION_ANGLE,
+        name="Oscillation Angle",
+        icon="mdi:angle-acute",
+        unit_of_measurement=DEGREE,
+        min_value=30,
+        max_value=140,
+        step=30,
+        method="async_set_oscillation_angle",
+    ),
+    FEATURE_SET_DELAY_OFF_COUNTDOWN: XiaomiMiioNumberDescription(
+        key=ATTR_DELAY_OFF_COUNTDOWN,
+        name="Delay Off Countdown",
+        icon="mdi:fan-off",
+        unit_of_measurement=TIME_MINUTES,
+        min_value=0,
```

Comment:
is 0 a valid value. Will it turn off the feature?

Reply:
Yes and yes.